mkdocs.yml

```diff
@@ -23,4 +27,5 @@ pages:
 # - Performance tips: performance.md
 - How it works: how-it-works.md
 - Class Reference: reference.md
-theme: readthedocs
+theme:
+  name: material
```

Comment:
It looks modern now :D

Reply:
Wasn't planning to change it, but my first try did error out with something related to the theme and i just copied their example. I like the style